lib/eggs/injected_app_bar.dart

```diff
@@ -54,7 +54,9 @@ class InjectedNavbar extends PBEgg implements PBInjectedIntermediate {
           .contains('<middle>')) {
         Interpret()
             .generateNonRootItem((node as PBInheritedIntermediate).originalRef)
-            .then((value) => middleItem = value);
```

Comment:
Did this make a functional difference or were we having difficulty reading this? @ivan-015 @SushiRoll53